dali/python/nvidia/dali/_utils/eager_utils.py

```diff
@@ -139,3 +198,444 @@ def _slice_tensorlist(data, size):
     """ Constructs TensorList consisting of ``size`` first elements of ``data``. """
 
     return type(data)(list(data)[:size], layout=data.layout())
+
+
+def _arithm_op(name, *inputs):
+    batch_size = _choose_batch_size(inputs)
+    inputs = [_Classification(input, f'Input {i}', arg_constant_len=batch_size).data
+              for i, input in enumerate(inputs)]
+    categories_idxs, inputs, integers, reals = _ops._group_inputs(
+        inputs, edge_type=(_tensors.TensorListCPU, _tensors.TensorListGPU))
+    input_desc = _ops._generate_input_desc(categories_idxs, integers, reals)
+    device = _ops._choose_device(inputs)
+
+    if device == "gpu":
+        inputs = list(input._as_gpu() if isinstance(
+            input, _tensors.TensorListCPU) else input for input in inputs)
+
+    init_args = {
+        'device': _ops._choose_device(inputs),
+        'expression_desc': f'{name}({input_desc})',
+        'integer_constants': integers,
+        'real_constants': reals,
+    }
+
+    from nvidia.dali.experimental.eager import arithmetic_generic_op
+    return arithmetic_generic_op(*inputs, **init_args)
+
+
+def _add(self, other):
```

Comment:
I asked for the docstring, because grepping with the leading underscore yields no results, so maybe the usage can be somehow explained?

Reply:
You mean that these implementations (`_add`, `_radd`, ...) appear to be unused? I added comment that they are used in the backend.